.gitignore

```diff
@@ -1,7 +1,19 @@
+# Own files
+output/
```

Comment:
This might not be a good idea if we want to save logs but ok

Reply:
I only save the 'prints' from Lisa in this folder. Usually I have a lot of shitty/not working output files in here which don't make sense to put on github

Reply:
Sure but the .gitignore file also applies to us. You should use https://stackoverflow.com/a/5724499 instead